h/static/styles/partials-v2/_form-input.scss

```diff
@@ -80,16 +80,31 @@
     right: $h-padding;
 
     color: $grey-5;
- }
+  }
+
+  .form-input__hint-icon {
+    display: none;
+    position: absolute;
```

Comment:
Using `absolute` positioning here means that the icon's positioning does not adapt to the length of the label, so this positioning is only correct for the label 'ORCID' and if the user agent is rendering the text with the same font metrics that you are testing with.

One way to fix this would be to put the icon inside the `<label>`